src/ExtEventLoop.php

```diff
@@ -178,8 +178,7 @@ public function tick()
 
         $this->futureTickQueue->tick();
 
-        // @-suppression: https://github.com/reactphp/react/pull/234#discussion-diff-7759616R226
-        @$this->eventBase->loop(EventBase::LOOP_ONCE | EventBase::LOOP_NONBLOCK);
+        $this->eventBase->loop(EventBase::LOOP_ONCE | EventBase::LOOP_NONBLOCK);
```

Comment:
I think we have 3 options now:

1. Keep the @-suppression
2. Use the workaround from my previous comment by temporarily set the error handler
3. Remove the @-suppression and document the change and it's impact in the changelog

My preference is option 3.